### PR TITLE
New Todo UI for FlightJS

### PR DESCRIPTION
**Pull request by kmargaritis**
Upgraded FlightJS to new Todo ui. 

* jasmine-flight and flightjs are linked to git repos

### CODE REVIEW

File: examples/flight/package.json [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
-    "karma-firefox-launcher": "~0.1.0",
-    "karma-phantomjs-launcher": "~0.1.0",
-    "karma-safari-launcher": "~0.1.1"
-  },
-  "scripts": {
-    "test": "karma start --browsers Firefox --single-run"
-  }
+	"name": "flight-todomvc",
+	"private": true,
+	"version": "0.0.1",
```
Comment (sindresorhus) [inline examples/flight/package.json]: `name` and `version` isn't needed

File: examples/flight/package.json [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
+	"dependencies": {
+		"depot": "~0.1.6",
```
Comment (sindresorhus) [inline examples/flight/package.json]: use `^` on all of these (instead of `~`).

File: examples/flight/bower.json [hunk 3]
```diff
@@ -2,15 +2,9 @@
...
-    "todomvc-common": "~0.3.0",
-    "requirejs-text": "~2.0.13"
+    "flight": "~1.3.0"
   },
   "devDependencies": {
-    "jasmine-flight": "~4.0.0",
-    "jasmine-jquery": "~2.0.5"
+    "jasmine-flight": "~4.0.0"
   }
 }
```
Comment (sindresorhus) [inline examples/flight/bower.json]: bower.json can be removed now

File: examples/flight/.gitignore [hunk 4]
```diff
@@ -1 +1,40 @@
...
+!node_modules/flightjs/lib
+
+node_modules/requirejs/*
+!node_modules/requirejs/require.js
+
+node_modules/requirejs-text/*
+!node_modules/requirejs-text/text.js
+
+node_modules/todomvc-app-css/*
+!node_modules/todomvc-app-css/index.css
```
Comment (sindresorhus) [inline examples/flight/.gitignore]: You need to actually include these files.

**Comment by kmargaritis** [comment]
This is exactly what happens when you are sending PRs seconds before leaving for a coffee!

File: examples/flight/.gitignore [hunk 5]
```diff
@@ -1 +1,40 @@
-node_modules/
+node_modules/.bin/*
+
+node_modules/depot/*
```
Comment (arthurvr) [inline examples/flight/.gitignore]: The start at the end isn't really needed. Ignoring the folder itself will work too.

**Comment by kmargaritis** [comment]
@sindresorhus I think it is ok. @arthurvr i didn't have time to remove the stars. Thanks for the heads up!

**Comment by kmargaritis** [comment]
@passy How can you check from Github which commits were not merged?

**Comment by kmargaritis** [comment]
@sindresorhus I saw you reverted this. I have updated the transient deps, put back the asterisks, and squashed the whole thing. Do you want to sent a new PR?

**Comment by sindresorhus** [comment]
Hmm, where did I revert this?

**Comment by kmargaritis** [comment]
[Here](https://github.com/tastejs/todomvc/commit/1e2fcbf1ba375fca43c398943e360a0e7f1db6ec)

**Comment by sindresorhus** [comment]
Oh, gawd... I didn't mean to push that commit. Fixed now.

**Comment by sindresorhus** [comment]
@kmargaritis Any idea why https://github.com/tastejs/todomvc/commit/c898a78d00ab80b4b2f91c45fa9fe4baebd3a482 contains so many changes?

**Comment by kmargaritis** [comment]
No idead @sindresorhus. Probably @passy has a clue. This is what i send [b6c18e2](https://github.com/tastejs/todomvc/commit/b6c18e2e8b602d2870f3018989ba250d2b475972)

**Comment by passy** [comment]
The commits are all in, but I squashed them before merging. I still have no idea what happened with that mega commit. Went through some theories with Sindre, but I'm still in the dark. Sorry for the poopoo.

**Pull request closed without merging**